config/opensearch_dashboards.yml

```diff
@@ -105,3 +105,9 @@
 # Specifies locale to be used for all localizable strings, dates and number formats.
 # Supported languages are the following: English - en , by default , Chinese - zh-CN .
 #i18n.locale: "en"
+
+# Set the allowlist to check input graphite url. 
+vis_type_timeline.graphiteUrls: ['https://www.hostedgraphite.com/UID/ACCESS_KEY/graphite']
```

Comment:
any reason why use this as default value and enable by default.

Reply:
By [#331](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/331) , allowlist and blocklist can't both be disabled. OSS sets allowlist as the default setting and provides a default value.

Reply:
better not set default value in config file. if do need a default value like any other configuration item, let's put in code.